PowerEditor/src/ScitillaComponent/FindReplaceDlg.h

```diff
@@ -410,15 +411,43 @@ class Progress
 	Progress(HINSTANCE hInst);
 	~Progress();
 
-	HWND open(HWND hOwner = NULL, const TCHAR* header = NULL);
-	bool isCancelled() const;
-	void setPercent(unsigned percent, const TCHAR *fileName) const;
-	void setInfo(const TCHAR *info) const {
+	HWND open(HWND hCallerWnd = NULL, const TCHAR* header = NULL);
+	void close();
+
+	bool isCancelled() const

```

Comment:
My intention was to make things a bit faster by making functions inline. For `isCancelled()` that's no big deal having in mind that it's being called 100 times total but `isCancelled()` is called a lot.